openjdk/build.xml

```diff
@@ -41,12 +41,29 @@
 				<available file="jtreg-4.2.0-tip.tar.gz" />
 			</not>
 			<then>
-				<exec executable="wget" failonerror="true">
-					<arg line="-q --no-check-certificate https://ci.adoptopenjdk.net/job/jtreg/lastSuccessfulBuild/artifact/jtreg-4.2.0-tip.tar.gz" />
-				</exec>
-				<exec executable="tar" failonerror="true">
-					<arg line="xf jtreg-4.2.0-tip.tar.gz -C ${DEST}" />
-				</exec>
+                            <if>
```

Comment:
Then this would end up looking more like what is in https://github.com/smlambert/openjdk-tests/blob/zos/openjdk/build.xml#L37-L57 once a patched jtreg executable can be available.

Reply:
yes